src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  @include flex($direction: column, $items: center);
+
+  box-sizing: border-box;
+  max-width: 200px;
+  width: 100%;
+  border: 1px solid $light-white-color;
+  border-radius: $default-border-radius;
+  background-color: $white-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    @include font(12px, 500, 18px);
```

Comment:
Verify that the `font` mixin is defined and imported correctly. This mixin is used to apply font styles, and any missing definition could lead to styling issues.